mlrun/datastore/v3io.py

```diff
@@ -118,3 +120,22 @@ def listdir(self, key):
 
         # todo: full = key, size, last_modified
         return [obj.key[subpath_length:] for obj in response.output.contents]
+
+
+def v3io_path(url):
```

Comment:
rename to something with a verb, `parse_v3io_path` ? `transform_v3io_path_to_url` ?
Also it seems that `url` should be renamed to `v3io_path`

Reply:
done